mp4parse_capi/src/lib.rs

```diff
@@ -417,6 +417,7 @@ pub unsafe extern fn mp4parse_get_track_info(parser: *mut mp4parse_parser, track
         TrackType::Unknown => return mp4parse_status::UNSUPPORTED,
     };
 
+    // Return UNKNOWN to gecko for unsupported format.
```

Comment:
Rather than reference Gecko explicitly (since this library is nominally disconnected from Gecko), just say something about not wanting to support these formats in the API?